api/v1beta1/openstackprovisionserver.go

```diff
@@ -25,7 +25,7 @@ func GetExistingProvServerPorts(
 	}
 
 	for _, provServer := range provServerList.Items {
-		found[provServer.Name] = provServer.Spec.Port
+		found[fmt.Sprintf("%s-%s", provServer.Name, provServer.Namespace)] = provServer.Spec.Port
```

Comment:
Should we use `namespacedName` type as key instead?
```
		namespacedName := types.NamespacedName{
			Name: provServer.Name, Namespace: provServer.Namespace }
		found[namespacedName.String()] = provServer.Spec.Port
```

Reply:
I had a look at this, so it would be handy if I already had a `types.NamespacedName` on the `instance`. But since I don't, I would need to import the apimachinery types here in this module which currently doesn't import it, then I need to build the type before referencing it. I'm not sure the extra step is adding a lot of value?

To avoid confusion, the latest update here stops referring to it as a NamespacedName